ricecooker/config.py

```diff
@@ -14,13 +14,78 @@
 SUSHI_BAR_CLIENT = None
 STAGE = False
 
-# Setup logger
+# Don't use this - call logging.getLogger(__name__) from each
+# individual module. Logging is configured centrally by calling
+# setup_logging()
 LOGGER = logging.getLogger()
 
-# Setup temporary stream handler (used until full setup in chef.config_logger)
-temporary_log_handler = logging.StreamHandler()
-LOGGER.setLevel(logging.INFO)
-LOGGER.addHandler(temporary_log_handler)
```

Comment:
This temporary stream handler needs to be present because many chef scripts will use config.LOGGER before actually calling `chef.main()` when the "real" logging is setup. Removing this temporary console printing handler loses such early log messages.

To test this try adding some logging messages to a sample chef script outside of the main class (e.g. at the begiining of the file).